src/libtrx/game/interpolation.c

```diff
@@ -473,9 +473,3 @@ void Interpolation_Interpolate(void)
         M_InterpolateBraid(m_WorldRate, lara_item);
     }
 }
-
-void Interpolation_RememberItem(ITEM *item)
-{
-    item->interp.prev.pos = item->pos;
-    item->interp.prev.rot = item->rot;
-}
```

Comment:
I hoped for a moment that not calling this in TR2 was the source of #2873, but bubbles are `EFFECT`s and here we talk `ITEM`s.